tardis/widgets/tests/test_line_info.py

```diff
@@ -0,0 +1,345 @@
+import pytest
+import pandas as pd
+import numpy as np
+from plotly.callbacks import Points, BoxSelector
+from tardis.widgets.line_info import LineInfoWidget
+from tardis.util.base import species_string_to_tuple
+
+
+@pytest.fixture(scope="class")
+def line_info_widget(simulation_verysimple):
+    line_info_widget = LineInfoWidget.from_simulation(simulation_verysimple)
+    return line_info_widget
+
+
+@pytest.mark.parametrize(
+    ("wavelength_range", "filter_mode"),
+    [
+        ([3000, 4000], "packet_out_nu"),
+        ([3000, 4000], "packet_in_nu"),
+        ([16200, 16300], "packet_out_nu"),
+        (None, "packet_in_nu"),
+    ],
+)
+class TestLineInfoWidgetData:
+    """Tests for methods that handles data in LineInfoWidget."""
+
+    def test_get_species_interactions(
+        self, line_info_widget, wavelength_range, filter_mode
```

Comment:
Can we add a description of exactly what this test is looking for and what constitutes a fail?